src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/services/CFTTaskMapper.java

```diff
@@ -85,18 +103,69 @@ public TaskResource mapToTaskObject(String taskId, List<TaskAttribute> taskAttri
             read(attributes, TASK_LOCATION_NAME, null),
             read(attributes, TASK_BUSINESS_CONTEXT, null),
             read(attributes, TASK_TERMINATION_REASON, null),
-            read(attributes, TASK_CREATED, null),
+            createdDate,
             read(attributes, TASK_ROLES, null)
         );
     }
 
+    private ExecutionTypeResource extractExecutionType(Map<TaskAttributeDefinition, Object> attributes) {
+        String executionTypeName = read(attributes, TASK_EXECUTION_TYPE_NAME, null);
+
+        if (executionTypeName != null) {
+            Optional<ExecutionType> value = ExecutionType.from(executionTypeName);
+            if (value.isPresent()) {
+                return new ExecutionTypeResource(
+                    value.get(),
+                    value.get().getName(),
+                    value.get().getDescription()
+                );
+            } else {
+                throw new IllegalStateException(
+                    "ExecutionTypeName value: '" + executionTypeName + "' could not be mapped to ExecutionType enum"
+                );
+            }
+        }
+        return null;
+    }
+
+    private List<NoteResource> extractWarningNotes(Map<TaskAttributeDefinition, Object> attributes) {
+        List<NoteResource> notes = null;
+        WarningValues warningList = read(attributes, TASK_WARNINGS, null);
+        if (warningList != null) {
+            List<Warning> warnings = warningList.getValues();
+            if (!warnings.isEmpty()) {
+                notes = warnings.stream()
```

Comment:
Can we add a test to cover this block ->
            if (!warnings.isEmpty()) {
                notes = warnings.stream()
                    .map(warning -> new NoteResource(
                        warning.getWarningCode(),
                        "WARNING",
                        null,
                        OffsetDateTime.now(),
                        warning.getWarningText()
                    )).collect(Collectors.toList());